CHANGELOG.md

```diff
@@ -5,11 +5,19 @@ It was first called Evopedia (and was using the file format of Evopedia).
 Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed to Kiwix-JS.
 
 ## Kiwix-JS v2.8.0
+
 Released on *TODO*
 
-Detailed changelog : TODO
+* NEW: Intelligently select the best Storage API for storing settings between sessions (`cookie` or `localStorage`)
```

Comment:
@mossroy I think we can add:

* NEW: User can change depth of title search from Configuration

If you agree I'll add it.

Reply:
Sure